frontend/src/components/MoviePageFilterByTitle.jsx

```diff
@@ -41,26 +46,38 @@ class MoviePageFilterByTitle extends React.Component {
     ).then(({ data }) => {
       this.setState({ castInfo: data });
     });
+    Axios.get(
+      `http://api.themoviedb.org/3/movie/${id}/videos?api_key=495d98b77df65d47fbf7eba028518ed7`
+    ).then(({ data }) => {
+      this.setState({ videoInfo: data });
+    });
   };
 
   render() {
-    //const {movieInfo} = this.state;
     const movieInfo = this.state.movieInfo;
+    const videoInfo = this.state.videoInfo;
+
     return (
       <div id="around">
         <Modal
           ariaHideApp={false}
           isOpen={this.state.modalIsOpen}
           className="modalStyle"
-          contentLabel="Example Modal"
-          onRequestClose={this.closeModal}
+          contentLabel="Trailer Modal"
+          onRequestClose={this.toggleModal}
         >
           <iframe
-            title="modal"
-            src={this.props.trailer}
+            title="Trailer"
+            src={`https://www.youtube-nocookie.com/embed/${videoInfo.results
+              .filter((video, i) => {
+                return i === 1;
+              })
+              .map((video, i) => {
+                return video.key;
+              })}`}
             frameBorder="0"
             allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-            allowFullScreen="1"
+            allowFullScreen
```

Comment:
is it normal if allowFullScreen is not defin now ?

Reply:
no :p 

Fixed